flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/handlers/TaskManagersHandler.java

```diff
@@ -53,6 +59,11 @@ public TaskManagersHandler(FiniteDuration timeout, MetricFetcher fetcher) {
 	}
 
 	@Override
+	public String[] getPaths() {
+		return new String[]{TASKMANAGERS_REST_PATH, TASKMANAGER_DETAILS_REST_PATH};
```

Comment:
yes, the same handler can handle both requests. The handlers that we created before were identical.